### DISCUSSION TITLE
Template .html error Capstone flask-ui

### DISCUSSION THREAD

**Discussion by rahmarania** [teamalgoritma/community#43] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/101410489/162574244-455fd7b5-8fe5-487f-971b-fe40977a1174.png)
sudah sesuai template tapi error

**Comment by onlyphantom** [comment, marked as answer]
Hello Rahma, you need to either close the `<img></img>` tag, or use a self-closing tag like `<img />`. Since `height` is not set, you can also omit it.

```
<img src="{{ results }}" width="700" />
```

**Reply by rahmarania** [reply]
![image](https://user-images.githubusercontent.com/101410489/162575552-5f139c15-e9bf-4f00-9a3b-b6062436c98b.png)
tag head nya masih error pak..

**Reply by onlyphantom** [reply]
It is not really an error, cuman warning doang. 😀

To fix it, add a `<title>My Web app</title>` on line 4, before the <Meta> tag. 

```
<html>
<head>
    <title>Rahma's Awesome Application</title>
    <meta charset="UTF-8">
    ...
</head>
```

**Reply by rahmarania** [reply]
warningnya sudah hilang tapi saya tidak bisa run project pakai python app.py, itu kenapa ya pak?

**Reply by onlyphantom** [reply]
Can you show me the error message you got? :)

**Reply by rahmarania** [reply]
![image](https://user-images.githubusercontent.com/101410489/162577874-2fceb1e1-33dd-49a1-91ae-c1154c9873de.png)
tidak ada error, baru bisa jalan(muncul url localhost) kalau pakai flask run
